python/python/bystro/prs/preprocess_for_prs.py

```diff
@@ -184,39 +195,41 @@ def generate_thresholded_overlap_scores_dosage(
     gwas_scores_path: str, dosage_matrix_path: str, p_value_threshold: float
```

Comment:
generate_thresholded_overlap_scores_dosage seems like an important function, could we add a test for it?